doc/proposals/improved-scorecard-config.md

```diff
@@ -0,0 +1,112 @@
+# Improved Scorecard Config
+
+Implementation Owner: AlexNPavel
+Status: Draft
+
+- [Background](#Background)
+- [Goals](#Goals)
+- [Design overview](#Design-overview)
+- [User facing usage](#User-facing-usage)
+
+## Background
+
+The `scorecard` subcommand in the `operator-sdk` currently has ~15 different command line flags. This creates
+unnecessary complexity for users. Also, many of the config options only apply to the internal plugins, and
+this may become confusing to users who use external plugins, as they may think that these config options
+affect their external plugins. Using a config file that configures plugins on an individual
+basis can help clarify these differences as well as make configuration a lot cleaner
+
+## Goals
+
+Drastically reduce the amount of command line flags in the `scorecard` subcommand and define a configuration file
+format that can allow for the simple configuration of both internal and external plugins.
+
+## Design overview
+
+The configuration for the scorecard will be in a `scorecard` subsection of the config file. This will allow
+the config file to continue working properly when the SDK adds global config file support for all subcommands.
+Under the scorecard subsection there are configuration options that apply to the entire scorecard as well as
+a section that allows for the configuration of both internal and external plugins. Here are the config options
+and what they do:
+
+- `kubeconfig` string - path to kubeconfig. This option sets the kubeconfig for internal plugins and sets the `KUBECONFIG` env var for external plugins
+- `output` string - sets output format. Valid values: `human-readable` and `json`
+- `plugin-dir` string - path to scorecard plugin directory. This is the directory where the plugins are run from, and all executable files in a `bin` subdirectory of the `plugin-dir` are automtically run by default.
+- `plugins` - an array of objects that configure both internal and external scorecard plugins
```

Comment:
Can you give an example of how this command would in full with the 3 elements mentioned bellow? Thanks!

Reply:
I'm not sure I understand your question.

Reply:
Sorry, how would I use the plugins option? `operator-sdk scorecard --plugins...` then what? :) Just want an example of the command with the other values you can specify, would user tend ot use all of them always. thanks!